vignettes/ggplot2-specs.Rmd

```diff
@@ -251,6 +251,8 @@ ggplot(sizes, aes(size, stroke, size = size, stroke = stroke)) +
   scale_size_identity()
 ```
 
+Because most shapes are not filled, mapping the `fill` aesthetic of points is considered an advanced feature. Additional steps may be required to produce sensible results, such as overriding the shape used in the `fill` guide (refer to `geom_point()` for an example of this).
+
```

Comment:
I don't think the first sentence clarifies anything.
I think it'd be better to just name the most common situation wherein the shape needs to be overriden, namely when you use a shape mapping with a discrete fill.